packages/amplify-util-mock/resources/mock-searchable-lambda-trigger/src/index.py

```diff
@@ -0,0 +1,256 @@
+import base64
```

Comment:
Is this the copy of open search streaming lambda?

Reply:
It is almost a copy. The changes that I did were:
1. Remove logic to make valid signed AWS SDK calls to interact with ElasticSearch cluster.
2. Remove the extra dependencies for AWS SDKs.
3. Pass the local cluster info as environment variables to call the local OpenSearch instance.

Reply:
Please add this context in a comment.